dali/test/python/test_fw_iterators.py

```diff
@@ -1480,3 +1484,28 @@ def test_paddle_prepare_first_batch():
     from nvidia.dali.plugin.paddle import DALIGenericIterator as PaddleIterator
     check_prepare_first_batch(PaddleIterator, output_map=["data"],
                               to_np=lambda x: np.array(x["data"]))
+
+@pipeline_def
+def feed_ndarray_test_pipeline():
+    return np.array([1], dtype=np.float)
+
+def test_mxnet_feed_ndarray():
+    from nvidia.dali.plugin.mxnet import feed_ndarray
+    import mxnet
+
+    pipe = feed_ndarray_test_pipeline(batch_size=1, num_threads=1, device_id=0)
+    pipe.build()
+    out = pipe.run()[0]
+    mxnet_tensor = mxnet.nd.empty([1], None, np.int8)
+    assert_raises(AssertionError, feed_ndarray, out, mxnet_tensor, glob="Type of DALI Tensor/TensorList doesn't match MXNet tensor type:")
```

Comment:
Update the pattern here if you update the message as suggested.

Reply:
Adjusted